### DISCUSSION TITLE
Release 1.9 on kirkstone

### DISCUSSION THREAD

**Discussion by paletteguy** [rauc/meta-rauc#262] [General]
Anybody working on getting rauc 1.9 on kirkstone ?

**Comment by ejoerns** [comment]
On my list for today, but not started, yet. Thus if you like, feel free :wink: 

Should be prepared (and merged) for master first, kirkstone can then be backported.

**Reply by paletteguy** [reply]
see master is only compatible with mickledore this is deliberate or can I add kirkstone ?

**Reply by ejoerns** [reply]
The main reason it's only master (mickledore) is that it's not actively tested against `kirkstone`.

**Reply by paletteguy** [reply]
ok, pr ready https://github.com/rauc/meta-rauc/pull/263

**Reply by paletteguy** [reply]
Anybody got the to review the PR: #263 ?

**Reply by ejoerns** [reply]
Merged

**Reply by paletteguy** [reply]
Backport to kirkstone.. 

PR: #264

**Reply by paletteguy** [reply]
PR: #264  Ready for review.
